pkg/kube/cluster-init.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/ash
```

Comment:
Why not make this /bin/sh? That is what we use in other scripts.

Reply:
ok